src/index.html

```diff
@@ -17,11 +17,12 @@
             </ul>
         </div>
     </header>
+    <section class="home"></section>
```

Comment:
Good job here, but I think it would be better if the 'main' tag is used instead of 'section' to follow the proper HTML semantic rule.